src/ch05-02-example-structs.md

```diff
@@ -225,42 +242,44 @@ struct Rectangle {
 fn main() {
     let rect1 = Rectangle { width: 30, height: 50 };
 
-    println!("rect1 is {:?}", rect1);
+    println!("rect1 to {}", rect1);
 }
 ```
 
-<span class="caption">Listing 5-12: Adding the annotation to derive the `Debug`
-trait and printing the `Rectangle` instance using debug formatting</span>
+<span class="caption">Listing 5-12: Dodanie adnotacji derywacji cechy `Debug`
+i wyświetlanie instancji `Rectangle` formatowaniem przeznaczonym do celów debugowania</span> 
 
-Now when we run the program, we won’t get any errors, and we’ll see the
-following output:
+Teraz kiedy uruchomimy program nie wyskoczy nam żaden błąd, a naszym oczom
+ukaże się poniższy tekst:
 
 ```text
-rect1 is Rectangle { width: 30, height: 50 }
+rect1 to Rectangle { width: 30, height: 50 }
```

Comment:
Tego nie wolno tłumaczyć, program wypisze "is"!

Reply:
169: zmieniłem `is` na `to`: `println!("rect1 to {}", rect1);`

zgodnie z sugestią zmian println-ów

Reply:
to samo w 267

Reply:
@beling daj lapke do gory jak jest ok, lapke w dol jak nie ok 🙏